src/net/if_.rs

```diff
@@ -213,26 +213,29 @@ libc_bitflags!(
         /// Interface is offline
         #[cfg(solarish)]
         IFF_OFFLINE;
-        #[cfg(target_os = "solaris")]
+        /*
+        // FIXME: The following are from 64 bits long long type.
```

Comment:
What's wrong here?  Do these not work correctly at runtime?  You should open a new issue for this and explain what the problem is.

Reply:
what do you have in mind ?

Reply:
> what do you have in mind ?

Cast them with `as c_int` if they are in the range `int`

Reply:
The thing is, for most of them it would overflow [from this one at least](https://github.com/rust-lang/libc/blob/497ac428bc010b5db9682ecf94cd567b31d53e5c/src/unix/solarish/mod.rs#L1891).

Reply:
> Can we cast them correctly?

Then we can not do this